libraries/pom.xml

```diff
@@ -385,7 +385,7 @@
         <rcaller.version>3.0</rcaller.version>
         <rserve.version>1.8.1</rserve.version>
         <agrona.version>1.17.1</agrona.version>
-        <exec-maven-plugin.version>1.6.0</exec-maven-plugin.version>
+        <!--<exec-maven-plugin.version>1.6.0</exec-maven-plugin.version>-->
```

Comment:
Why do we need to comment this out?

Reply:
@alvin577austria the parent of this module already has the latest version of 3.1.0. I was just checking if that will work and if we are getting build errors if we use parent updated version.
I am removing this as this is working fine.